lib/puppet/application/device.rb

```diff
@@ -372,9 +377,9 @@ def find_resources(type, name)
     end
   end
 
-  def setup_host(name)
+  def setup_host
     waitforcert = options[:waitforcert] || (Puppet[:onetime] ? 0 : Puppet[:waitforcert])
-    sm = Puppet::SSL::StateMachine.new(certname: name, waitforcert: waitforcert)
```

Comment:
https://github.com/puppetlabs/puppet/blob/85be99da5bb4cdd011ee68b00746ec2e9b6e65fc/lib/puppet/ssl/state_machine.rb#L210-L213
It looks like nothing was ever done with `certname` in the statemachine

This comment is mostly for my peace of mind, knowing nothing will be impacted by not including this var in this method call

Reply:
An earlier version of the client state machine accepted a `:certname` keyword argument, but I took it out, and forgot to fix `device.rb`.